pyproject.toml

```diff
@@ -1,7 +1,35 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]~=6.0"]
+requires = ["setuptools>=64", "wheel", "setuptools_scm[toml]>=8"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "diepvries"
+dynamic = ["version"]
+description = "diepvries - Picnic Data Vault framework"
+readme = {file = "README.md", content-type = "text/markdown"}
+keywords = ["data vault"]
+license = {file = "LICENSE"}
+authors = [{name = "Picnic Technologies"}]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+]
+dependencies = ["snowflake-connector-python~=3.0"]
+requires-python = ">= 3.8"
+
+[project.optional-dependencies]
+test = ["pytest~=6.2", "tox"]
```

Comment:
Should `tox` be here?

Reply:
True, I guess it's a better fit in a `dev` group. I do think it's nice to have.